src/main/java/com/ecwid/maleorang/method/v3_0/lists/abuse_reports/AbuseReportInfo.kt

```diff
@@ -0,0 +1,91 @@
+package com.ecwid.maleorang.method.v3_0.lists
+
+import com.ecwid.maleorang.MailchimpObject
+import com.ecwid.maleorang.annotation.Field
+
+/**
+ * Created by apocheau on 27/08/16.
+ */
+class   AbuseReportInfo : MailchimpObject() {
+
+    @JvmField
+    @Field
+    var id: String? = null
+
+    @JvmField
+    @Field
+    var campaign_id: String? = null
+
+    @JvmField
+    @Field
+    var list_id: String? = null
+
+    @JvmField
+    @Field
+    var email_id: String? = null
+
+    @JvmField
+    @Field
+    var email_address: String? = null
+
+    @JvmField
+    @Field
+    var merge_fields: Object? = null
+
+    @JvmField
+    @Field
+    var vip: Boolean? = null
+
+    @JvmField
+    @Field
+    var date: String? = null
+
+    @JvmField
+    @Field
+    var _links: List<LinkInfo>? = null
+
+    fun setId(id: String): AbuseReportInfo{
```

Comment:
Yes it was made for chaining methods and build object in test method more easily. For example:
`AbuseReportInfo().setId("2345").setCampaign_id("6789");`
The "Kotlin way" is also nice and i didn't know it, thanks. I will follow it.